cases/scenarios/ts-pre-modules/scenario.json

```diff
@@ -4,8 +4,10 @@
     "args": [
         "-p",
         "${suiteDirectory}/ts-pre-modules/src/compiler",
-        "--outdir",
-        "${outDirectory}"
+        "--outfile",
+        "${outDirectory}/out.js",
+        "--composite",
```

Comment:
Do you want to add --declaration so that it still does same thing as generating declaration but not incrementally ?

Reply:
Dont bother.. its already there in the config

Reply:
I can, but it may not actually matter since this benchmark has errors and so wouldn't emit, right?